src/ncp/ncp_base.cpp

```diff
@@ -202,6 +239,31 @@ static spinel_status_t ResetReasonToSpinelStatus(otPlatResetReason aReason)
 
 NcpBase *NcpBase::sNcpInstance = nullptr;
 
+#if OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE && OPENTHREAD_RADIO
+NcpBase::NcpBase(Instance **aInstances, uint8_t aCount)
+    : NcpBase(aInstances[0])
+{
+    OT_UNUSED_VARIABLE(aCount);
+    OT_ASSERT(aCount > 0);
+    OT_ASSERT(aCount < SPINEL_HEADER_IID_MAX + 1);
+
+    uint8_t skipped = 0;
+
+    for (int i = 0; i < aCount; i++)
```

Comment:
If the broadcast `iid` set to zero with `aCount` to 2 then `for` loop will exit without processing the last `aInstances`.